tests/test_99_3Projects.py

```diff
@@ -113,12 +108,10 @@ def test_main(self):
 
       ###  TEST MEASUREMENTS AND SCANNING
       outputString(outputFormat,'h2','TEST MEASUREMENTS AND SCANNING')
-      examplePath = Path(__file__).parent/'ExampleMeasurements'
-      shutil.copy(examplePath/'Zeiss.tif', semDirName)
```

Comment:
I still see these files committed, if not used else, can we delete them from repo too?

Reply:
I don't understand as they are used in the example data created after installation and they are used in tests, which do not run on github but on your local installation

Reply:
I corrected that